akka-contrib/src/main/scala/akka/contrib/circuitbreaker/askExtensions.scala

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.contrib.circuitbreaker
+
+import akka.actor.{ ActorSelection, Actor, ActorRef }
+import akka.contrib.circuitbreaker.CircuitBreakerProxy.CircuitOpenFailure
+import akka.util.Timeout
+import scala.language.implicitConversions
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+final class OpenCircuitException extends RuntimeException("Unable to complete operation since the Circuit Breaker Actor Proxy is in Open State")
```

Comment:
I wasn't sure how the Java compatibility of that would have been. I am thinking to create a  `OpenCircuitException` as a single instance extending the class `OpenCircuitException` to be used by the Java code. Would that be good? or is it useless?